.github/workflows/test.yml

```diff
@@ -0,0 +1,82 @@
+name: Python tests
+
+# This file tests the claimed support range of PyBIDS including
+#
+# * Operating systems: Linux, OSX
+
+on:
+  push:
+    branches:
+      - master
+      - maint/*
+  pull_request: {}
+  schedule:
+    # 8am EST / 9am EDT Mondays
+    - cron: '0 13 * * 1'
+  # Allow job to be triggered manually from GitHub interface
+  workflow_dispatch:
+
+defaults:
+  run:
+    shell: bash
+
+# Force tox and pytest to use color
+env:
+  FORCE_COLOR: true
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    # Check each OS, all supported Python, minimum versions and latest releases
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: ['ubuntu-latest', 'macos-latest']
```

Comment:
no windows?

Reply:
We haven't tested on Windows in a long time. If someone wants to add it, they may. I don't want to take this PR down that rabbit hole.